readme.md

```diff
@@ -1,16 +1,30 @@
-# WeePickle [![Build Status](https://travis-ci.org/rallyhealth/weePickle.svg)](https://travis-ci.org/rallyhealth/weePickle) <img src="https://api.bintray.com/packages/rallyhealth/maven/weePickle/images/download.svg" title="Download" />
+# weePickle [![Build Status](https://travis-ci.org/rallyhealth/weePickle.svg)](https://travis-ci.org/rallyhealth/weePickle) [![sbt instructions](https://api.bintray.com/packages/rallyhealth/maven/weePickle/images/download.svg)](#sbt)
 
-A JSON, YAML, MsgPack, XML, etc. serialization framework with MiMa and shading.
+A *stable* JSON, YAML, MsgPack, XML, etc. serialization framework based on [uPickle](https://lihaoyi.github.io/upickle).
 
-Safe for use in libraries without causing dependency hell ([rationale](shading.md)).
+## We're tired of dependency hell!
+weePickle exists to fulfill two promises:
+
+1. We will not break compatibility in `v1.x.y`, as enforced by [MiMa](https://github.com/lightbend/mima).
+2. When we release `v2.0.0`, you can use it immediately **without waiting for your other library dependencies to update**. We achieve this by [shading](https://github.com/rallyhealth/sbt-shading).
+
+### Shading
+Both `weepickle-v1.jar` and `weepickle-v2.jar` (in the future) will coexist on the classpath peacefully by applying [shading](https://github.com/rallyhealth/sbt-shading) at multiple levels.
+1. All artifact names are suffixed with `-v1`, which prevents evictions.
+2. All packages are prefixed with `com.rallyhealth.v1`, which prevents classpath conflicts.
+
+Shading allows libraries to depend directly on [weePickle-v1](#sbt) without fear of causing incompatible evictions and runtime failures.
+
+For more background, see [shading.md](shading.md).
 
 ## Features
-- [SemVer](https://semver.org/) ([mima](https://github.com/lightbend/mima)) + [Shading](https://github.com/rallyhealth/sbt-shading) => strong backwards compatibility + forwards interop, even across major versions
-- [Zero-overhead conversion](http://www.lihaoyi.com/post/ZeroOverheadTreeProcessingwiththeVisitorPattern.html) between:
-    - [jackson-core](https://github.com/FasterXML/jackson#active-jackson-projects) (JSON, YAML, XML, CBOR, SMILE, Ion, etc.)
-    - scala json ASTs (circe, json4s, play-json, argonaut)
-    - case classes (flexible macros)
-    - [MessagePack](#messagepack)
+weePickle combines some of the best parts of the serialization ecosystem.
+
+- [Zero-overhead conversion of uPickle](http://www.lihaoyi.com/post/ZeroOverheadTreeProcessingwiththeVisitorPattern.html)
+- Async parsing of [jackson-core](https://github.com/FasterXML/jackson#active-jackson-projects), and broad format support (JSON, YAML, XML, CBOR, SMILE, Ion, etc.)
```

Comment:
to
```
[and broad format support (JSON, YAML, XML, CBOR, SMILE, Ion, etc.)](readme.md#supported-types)
```

Reply:
jackson-core only supplies one of those. I agree it's muddled. I've tweaked it slightly.